frontend_tests/node_tests/settings_user_groups.js

```diff
@@ -79,7 +79,7 @@ test_ui("can_edit", () => {
 });
 
 const user_group_selector = `#user-groups #${CSS.escape(1)}`;
-const cancel_selector = `#user-groups #${CSS.escape(1)} .save-status.btn-danger`;
+const cancel_selector = `#user-groups #${CSS.escape(1)} .undo-button.btn-danger`;
```

Comment:
The Undo button is still functional for canceling the changes in members input area. Shall i remove the undo button for adding/deleting of members?

Reply:
Ohh, ok we can keep undo button but am not sure since we do not keep it in other places using user pills like stream subscribers list. we can wait for more feedback before doing any change here.

Reply:
I think undo button has almost no use case. The user can still remove the member by clicking on the respective ```x``` button which is more precise than clicking the undo button. Also i haven't seen undo functionality anywhere else in Zulip.